commands.json

```
@@ -1042,6 +1042,12 @@
     "since": "3.0.0",
     "group": "cluster"
   },
+  "CLUSTER LINKS": {
+    "summary": "Returns a list of all TCP connections to and from peer nodes in cluster",
```

```suggestion
    "summary": "Returns a list of all TCP links to and from peer nodes in cluster",
```
You use link throughout the second document.